docs/content/demos/desktop-room-live.sh

```
@@ -0,0 +1,118 @@
+  --name avatar --name "${AGENTS[0]}" --name "${AGENTS[1]}" \
+  --out-dir "$E2E/identities" >"$E2E/derive.log" 2>&1
+
+did_of() {
+  python3 -c "import json,sys; print(json.load(open(sys.argv[1]))['id'])" \
+    "$E2E/identities/$1.did.json"
+}
+DIDS="$(did_of avatar)"
+for a in "${AGENTS[@]}"; do DIDS="$DIDS,$(did_of "$a")"; done
+
+export SHADI_SLIM_AUTH=did
+export SLIM_HUMAN_SEED="$(cat "$E2E/human-seed.txt")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)